packages/block-library/src/post-template/style.scss

```
@@ -4,6 +4,7 @@
 	max-width: 100%;
 	list-style: none;
 	padding: 0;
+	box-sizing: border-box;
```

```suggestion
	// This block has customizable padding, border-box makes that more predictable.
	box-sizing: border-box;
```

As with any block, it's a good idea to add comments.